src/notifications/screens/notifications.screen.js

```diff
@@ -147,6 +165,16 @@ class Notifications extends Component {
     this.props.getAllNotifications();
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (
+      !nextProps.isPendingMarkAllNotificationsAsRead &&
+      this.props.isPendingMarkAllNotificationsAsRead &&
+      !this.isLoading()
+    ) {
+      this.getNotifications()();
```

Comment:
()() ?

Reply:
Because this function returns a function, it still needs to be called, so one more parentheses.

Reply:
How about rename `getNotifications` as `getFuncForGettingNotifications`, or something else?

Reply:
It is unlikely that the first name is fine (`getFuncForGettingNotifications`), because this function does two things: not only returns another function, but also updates the notification counter.

Reply:
@lex111 Sorry, I didn't quite get your words. Can you tell me where "updates the notification counter"?

Reply:
It's you who excuse me :smile: , I mixed up with my other PR #360  - I'm doing a notification counter there and there's [this](https://github.com/gitpoint/git-point/blob/f522c6bc7781864083418a858174d5026ed4f6c2/src/notifications/screens/notifications.screen.js#L171) function.
But we need to think more, we can be renamed.

Reply:
I think something like `getNotificationsForCurrentTab` or something along those lines will be okay

Reply:
@lex111 I prefer to keep `getNotifications` as simple as now, instead of mixing `getNotificationsCount` in it.

Because there is [another place](https://github.com/gitpoint/git-point/blob/f522c6bc7781864083418a858174d5026ed4f6c2/src/notifications/screens/notifications.screen.js#L397) calls this function, which just expects a function. After #360 is merged, I guess `getNotificationsCount` will be called frequently by `render`.